src/lib/stores/Quaff.ts

```diff
@@ -1,9 +1,10 @@
 import { writable, derived } from "svelte/store";
 import { page } from "$app/stores";
+import version from "$lib/helpers/version";
 
 function quaff() {
   const { subscribe, set, update } = writable({
-    version: __QUAFF_VERSION__,
```

Comment:
The Quaff version should still be available on the $Quaff object, do you already have an idea how to get it?